README.md

```diff
@@ -27,6 +29,8 @@ It will listen on <http://127.0.0.1:3030> by default.
 
 Note: If you have issues with `npm install` please update to npm 2.0+ using `npm install -g npm@2` 
 ([Issue #1594](https://github.com/mozilla/fxa-content-server/issues/1594))
+Note: If you have issues with bigint (gmp.h not found) that stops pm2 from being installed, install pm2
```

Comment:
Not sure about this note. 1. pm2 is not used with the content server itself. 2. this is not a valid suggestion